### PR TITLE
#74 chore: update README.md

### PR DESCRIPTION
**Pull request by AjithLalps**
Updated the readme file to include file upload instead of using direct image links. fix for the issue #74

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +136,58 @@ export default {
 </script>
 ```
 
+### File Upload Sample
+You can upload file instead of using direct image link.
+*Usage*
+In your form add a `file input` along with `vue-croppie` component.
+```html
+<input :class="form-control"  type="file" @change="croppie($event)/>
```
Comment (jofftiquez) [inline README.md]: Why is the class using data binding? `:class="form-control"` The class is unnecessary for an example, especially that this class is from bootstrap I presume?

Comment (jofftiquez) [reply README.md]: In the `@change` event. Please do just `@change="croppie"` the event object is being passed implicitly in the method so you can just get the event in the method declaration.

Comment (jofftiquez) [reply README.md]: The `@change="croppie($event)` is missing a closing quote.

File: README.md [hunk 2]
```diff
@@ -136,6 +136,58 @@ export default {
...
+ <vue-croppie ref="croppieRef" :enableOrientation="true" :boundary="{ width: 450, height: 300}" :viewport="{ width:400, height:250, 'type':'square' }">
+ </vue-croppie>
+<!-- the result -->
+<img v-bind:src="cropped">
```
Comment (jofftiquez) [inline README.md]: Change `v-bind:src` to shorthand version `:src`

File: README.md [hunk 3]
```diff
@@ -136,6 +136,58 @@ export default {
...
+<button @click="crop()">Crop</button>
+```
+and add script to manage file upload and binding result to expected variable.
```
Comment (jofftiquez) [inline README.md]: Why is this formatted this way?

File: README.md [hunk 4]
```diff
@@ -136,6 +136,58 @@ export default {
...
+```vue
+<script>
+export default {
+	data() {
+		return {
+			croppie_image: "",
```
Comment (jofftiquez) [inline README.md]: Use single quote

File: README.md [hunk 5]
```diff
@@ -136,6 +136,58 @@ export default {
...
+		};
+	},
+	methods: {
+		croppie(e) {
```
Comment (jofftiquez) [inline README.md]: add space before parens `croppie [space] (e)`  so `croppie (e) {`

Comment (jofftiquez) [reply README.md]: Apply the same to other methods

File: README.md [hunk 6]
```diff
@@ -136,6 +136,58 @@ export default {
...
+			reader.onload = e => {
+				this.$refs.croppieRef.bind({
+					url: e.target.result
+				});
+			};
+
+			reader.readAsDataURL(files[0]);
+		},
+		crop() {
+    //options can be updated.
```
Comment (jofftiquez) [inline README.md]: Use proper grammar, capitalize first letter for all sentences.

File: README.md [hunk 7]
```diff
@@ -136,6 +136,58 @@ export default {
...
+    // current option will return a base64 version of the uploaded image with a size of 600px X 450px.
+			let options = {
```
Comment (jofftiquez) [inline README.md]: Use const if the value will not be amended anywhere.

File: README.md [hunk 8]
```diff
@@ -136,6 +136,58 @@ export default {
...
+				type: "base64",
```
Comment (jofftiquez) [inline README.md]: Use single quote. Apply to JS.

File: README.md [hunk 9]
```diff
@@ -136,6 +136,58 @@ export default {
...
+				size: { width: 600, height: 450 },
+				format: "jpeg"
+			};
+			this.$refs.croppieRef.result(options, output => {
+				this.croppie_image = output;
```
Comment (jofftiquez) [inline README.md]: Use camelCase. Apply to all.

**Review by jofftiquez — CHANGES_REQUESTED** [review]

**Comment by jofftiquez** [comment]
Hello @AjithLalps kindly see review comments. Thanks for the PR.

File: README.md (hunk 3 above)
Comment (AjithLalps) [reply README.md]: Then how to format this?

File: README.md (hunk 7 above)
Comment (AjithLalps) [reply README.md]: I didn't get you

**Comment by AjithLalps** [comment]
Hi @jofftiquez,

 All changes mentioned have been updated.

Please review it and let me know if there is anything to be updated.

 Thank you

File: README.md (hunk 3 above)
Comment (AjithLalps) [reply README.md]: To separate html and vue code

**Review by jofftiquez — CHANGES_REQUESTED** [review]
Sorry, I viewed the output. I think it would be better to put it in a single file component link the original sample. Also fix indention of code coz it looks kinda weird.

Screen Shot 2020-02-19 at 1 54 24 PM

**Comment by AjithLalps** [comment]
Formatting was my mistake. updated. Please check

**Review by jofftiquez — APPROVED** [review]

File: README.md [hunk 10]
```diff
@@ -136,6 +136,59 @@ export default {
...
+  <img :src="cropped">
+  <button @click="crop">Crop</button>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      croppieImage: '',
+      cropped: null,
```
Comment (jofftiquez) [inline README.md]: Please remove dangling comma

File: README.md [hunk 11]
```diff
@@ -136,6 +136,59 @@ export default {
...
+      reader.onload = e => {
+        this.$refs.croppieRef.bind({
+          url: e.target.result
+        });
+      };
+
+    reader.readAsDataURL(files[0]);
+    },
+    crop() {
+      //Options can be updated.
```
Comment (jofftiquez) [inline README.md]: Put space after //

**Comment by jofftiquez** [comment]
Hello just fix the few minor things, and it's good to go.

File: README.md [hunk 12]
```diff
@@ -136,6 +136,59 @@ export default {
...
+      // Options can be updated.
+      // Current option will return a base64 version of the uploaded image with a size of 600px X 450px.
+      let options = {
+        type: 'base64',
+        size: { width: 600, height: 450 },
+        format: 'jpeg'
+      };
+      this.$refs.croppieRef.result(options, output => {
+        this.cropped = this.croppieImage = output;
+          console.log(this.croppie_image);
```
Comment (jofftiquez) [inline README.md]: This variable was not declared.

**Review by jofftiquez — CHANGES_REQUESTED** [review]

**Review by jofftiquez — APPROVED** [review]

**Comment by jofftiquez** [comment]
Thanks @AjithLalps

**Pull request merged**